app/views/users/_sidebar.html.erb

```diff
@@ -56,6 +56,13 @@
             <%= pluralize @user.decorate.cached_followed_tags.size, "Tag" %> Followed
           </div>
         </div>
+        <div class="daily-stats">
```

Comment:
I like that you made the tags within this div semantic (`h3`, `p`). Will follow suite when working on the div for overall stats.